CMakeLists.txt

```diff
@@ -1385,6 +1392,9 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/cmake_templates/zeek-config-paths.h.i
                ${CMAKE_CURRENT_BINARY_DIR}/zeek-config-paths.h)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/zeek-config-paths.h DESTINATION include/zeek)
 
+zeek_finalize_bif_autogen_stubs(src/__all__.bif.cc src/__all__.bif.init.cc
```

Comment:
Maybe add a comment here why this is the right place to call this? I couldn't immediately tell which of the above make any modifications.